packages/lab/src/radio-button/RadioButtonIcon.css

```diff
@@ -1,84 +1,76 @@
-.salt-density-high {
-  --radioButton-icon-size: 12px;
-}
-
-.salt-density-medium {
-  --radioButton-icon-size: 14px;
-}
-
-.salt-density-low {
-  --radioButton-icon-size: 16px;
-}
-
-.salt-density-touch {
-  --radioButton-icon-size: 18px;
-}
-
 /* Styles applied to root component */
 .saltRadioButtonIcon {
   --radioButton-icon-fill: transparent;
   --radioButton-icon-fill-checked: var(--salt-selectable-borderColor-selected);
   --radioButton-icon-fill-checked-disabled: var(--salt-selectable-borderColor-selectedDisabled);
+  --radioButton-icon-fill-error: var(--salt-palette-error-border);
```

Comment:
```suggestion
  --radioButton-icon-fill-error: var(--salt-status-error-border);
```

Reply:
i'll make these changes after design spec is ready

Reply:
done. I'll change again if needed